nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3527,7 +3527,7 @@ For example, to provide two additional network interfaces, a user could also spe
  +
 Providing three total network interfaces, including  `nifi.web.http.network.interface.default`.
 |`nifi.web.https.host`|The HTTPS host. It is blank by default.
```

Comment:
Should change the language for nifi.web.https.host since it's no longer blank by default.  Reverse comment for nifi.web.http.host.

Reply:
Thanks for catching that detail, I pushed an update with changes.